### DISCUSSION TITLE
Security Audit of cQube v5.0

### DISCUSSION THREAD

**Discussion by ananya-jain1** [Sunbird-cQube/community#765] [General]
Hey folks,
We will be initiating the security audit of cQube v5.0 application. It is necessary so that multiple states and central agencies can adopt cQube seamlessly without any security-related concern. 

The security audit will help protect critical data, identify security loopholes, ensure compliance with regulations, create new security policies and track the effectiveness of security strategies.

A scope of work has been created for the security audit of cQube v5.0 [here](https://project-sunbird.atlassian.net/wiki/spaces/CQUB/pages/3297739307/cQube+v5.0+Scope+of+Work+for+Security+Audit). 

It would be great if the community members can help us in defining the scope further through detailed reviews, before we take it to security audit by an external agency. Looking forward to hearing from some of the members!

@surendrasinghs @sukhpreetsamagra @rahul101001000 @sagarihanda @ChakshuGautam @Shruti3004 @vijayatgh

**Comment by surendrasinghs** [comment]
@gandham-santhosh @SanthoshVasabhaktula @mathewjpallan request your support in this.

**Comment by rhwarrier** [comment]
@mathewjpallan @anandp504 @sajeshkayyath  
The idea is to eventually do a third party audit. However, in order to ensure we're better prepared for it, we'd like to do an internal evaluation first - would you be able to do a quick review / are there specific questions that we should look at answering ?
All help would be appreciated, thanks.
Please let us know if there are any specific inputs you require from the team.

**Comment by DevashishRaghav** [comment]
[Security audit April 2023.pdf](https://github.com/Sunbird-cQube/community/files/12068248/Security.audit.April.2023.pdf)
Please also find attached the security audit report that was done last. All the issues highlighted here have been fixed the current version. Hope this helps to expedite the process
